client/go/vespa/load_env_test.go

```diff
@@ -98,3 +99,59 @@ override VESPA_V2 v2
 	assert.NotNil(t, err)
 	assert.Equal(t, err.Error(), "Not a valid environment variable name: '.A'")
 }
+
+func TestFindUser(t *testing.T) {
+	u := FindVespaUser()
+	if u == "" {
+		fmt.Fprintln(os.Stderr, "WARNING: empty result from FindVespaUser()")
+	} else {
+		fmt.Fprintln(os.Stderr, "INFO: result from FindVespaUser() is", u)
+		assert.Equal(t, u, os.Getenv("VESPA_USER"))
+	}
+	setup(t, `
+override VESPA_USER unprivuser
+`)
+	LoadDefaultEnv()
+	u = FindVespaUser()
+	assert.Equal(t, "unprivuser", u)
+}
+
+func TestExportEnv(t *testing.T) {
+	os.Setenv("VESPA_FOO", "was foo")
```

Comment:
Consider using `t.Setenv` which automatically clears the variables after this test finishes.